Agents/ForecastingAgent/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.8"
 
 services:
   forecasting_agent:
-    image: ghcr.io/cambridge-cares/forecasting_agent:1.1.0
+    image: ghcr.io/cambridge-cares/forecasting_agent:1.1.1
```

Comment:
If you update the version number here, then please don't forget to update the version number in `/ForecastingAgent/stack-manager-input-config/forecasting-agent.json`.

Reply:
Addressed by [c486fb9](https://github.com/cambridge-cares/TheWorldAvatar/pull/798/commits/c486fb9b1517ec6e9770a0f2a5d59e45a773bb0f) and will push docker image after successful merge to `main`